server/controllers/order.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-console */
+const addOrder = require('../database/queries/order');
+const addMeal = require('../database/queries/orderMeal');
+
+module.exports = (req, res, next) => {
+  const {
```

Comment:
you should destructure the values out of the body directly before validating them

Use `joi` to create a schema and validate the `req.body` values